src/logistration/tests/RegistrationPage.test.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import configureStore from 'redux-mock-store';
+import { IntlProvider, injectIntl, configure } from '@edx/frontend-platform/i18n';
+
+import RegistrationPage from '../RegistrationPage';
+
+const IntlRegistrationPage = injectIntl(RegistrationPage);
+const mockStore = configureStore();
+
+describe('./RegistrationPage.js', () => {
+  const initialState = {
+    logistration: {
+      registrationResult: { success: false, redirectUrl: '' },
+    },
+  };
+
+  let props = {};
+  let store = {};
+
+  const reduxWrapper = children => (
+    <IntlProvider locale="en">
+      <Provider store={store}>{children}</Provider>
+    </IntlProvider>
+  );
+
+  beforeEach(() => {
+    configure({
+      loggingService: { logError: jest.fn() },
+      config: {
+        ENVIRONMENT: 'production',
+        LANGUAGE_PREFERENCE_COOKIE_NAME: 'yum',
+      },
+      messages: {
+        'es-419': {},
+        de: {},
+        'en-us': {},
+      },
+    });
```

Comment:
Why we need all this here?

Reply:
we need i18 configuration because we are using `getLocale and getCountryList ` methods. If we don't add the configuration it would throw the error getLocale called before the configure.